src/components/carousel/carousel.scss

```diff
@@ -0,0 +1,68 @@
+.carousel {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    overflow: hidden;
+
+    &__container {
+        display: flex;
+        height: 100%;
+        width: 300%;
+
+        &__child {
+            width: 100%;
```

Comment:
You don't need it, it will be taken from your image width

Reply:
I think i need to define width for __child div. If i do not, then all there pages will be shown at the same time. However, here i define width 100% for this div and image, and at a moment, only one image is shown.

Reply:
You can remove 100% now I think